enterprise/internal/codeintel/lsifserver/client/client.go

```diff
@@ -1,21 +1,43 @@
 package client
 
 import (
+	"errors"
 	"net/http"
+	"strings"
+	"sync"
 
 	"github.com/opentracing-contrib/go-stdlib/nethttp"
-	"github.com/sourcegraph/sourcegraph/enterprise/internal/codeintel/lsifserver"
+	"github.com/sourcegraph/sourcegraph/internal/endpoint"
+	"github.com/sourcegraph/sourcegraph/internal/env"
 )
 
-var DefaultClient = &Client{
-	URL: lsifserver.ServerURLFromEnv,
-	HTTPClient: &http.Client{
-		// nethttp.Transport will propagate opentracing spans
-		Transport: &nethttp.Transport{},
-	},
-}
+var (
+	lsifServerURL = env.Get("LSIF_SERVER_URL", "k8s+http://lsif-server:3186", "lsif-server URL")
```

Comment:
```suggestion
	lsifServerURL = env.Get("LSIF_SERVER_URL", "k8s+http://lsif-server:3186", "lsif-server URL (or space separated list of lsif-server URLs)")
```

Slightly better docs here, since `endpoint.New` does take the space separated list: https://sourcegraph.com/github.com/sourcegraph/sourcegraph@f3e5cccad5dd68cd8360bbe571a193601dce833e/-/blob/internal/endpoint/endpoint.go#L32-49

Naming is still OK since everything else follows a similar pattern for legacy reasons (i.e. `SEARCHER_URL` is singular despite also allowing multiple searcher URLs to be specified)